pytrinamic/ic/TMC5240.py

```diff
@@ -0,0 +1,323 @@
+from ..ic.tmc_ic import TMCIc
+from ..features.motor_control_ic import MotorControlIc
+
+
+
+class TMC5240(TMCIc):
+    """
+        The TMC5240 is a smart high-performance stepper motor controller and driver IC with serial communication interfaces (SPI, UART).
+        Supply voltage: 4,5-36V
+    """
+    def __init__(self, parent_eval):
+        super().__init__("TMC5130", self.__doc__)
```

Comment:
There is a remaining item from the 5130, you could do this
```python
super().__init__(self.__class__.__name__, self.__doc__)
```
So this mistake should not happen again if someone is making a copy of this file. [https://www.geeksforgeeks.org/python-program-to-get-the-class-name-of-an-instance/](https://www.geeksforgeeks.org/python-program-to-get-the-class-name-of-an-instance/)
Not sure why we do that at all, haha.